devtools/org.eclipse.xsemantics.workspace/Xsemantics.setup

```diff
@@ -189,7 +189,7 @@
       <requirement
           name="org.eclipse.b3.aggregator.editor.feature.feature.group"/>
       <repository
-          url="http://download.eclipse.org/modeling/emft/b3/updates-4.4"/>
+          url="http://download.eclipse.org/cbi/updates/aggregator/ide/4.4/?d"/>
```

Comment:
I think that the "?d" is wrong: it's useful only for browsing the contents via web but it does not work with p2. I also think that for Oxygen the "4.4" should be increased... In any case, I'm not using the aggregator with Xsemantics anymore, so we could also remove such an installed feature from the Oomph setup